sycl/plugins/unified_runtime/pi2ur.hpp

```diff
@@ -242,6 +241,7 @@ namespace pi2ur {
 inline pi_result piPlatformsGet(pi_uint32 num_entries, pi_platform *platforms,
                                 pi_uint32 *num_platforms) {
 
+  urInit(0);

```

Comment:
I think this should go inside `urPlatformGet`

Reply:
urInit has to be called before any other ur api

Reply:
then that should be called from SYCL RT directly, no?

Reply:
Do you have a suggestion for where to put it? Some initialization path somewhere?

Reply:
@smaslov-intel : I think this place is correct. piPlatformsGet is typically the first function to be called, and then this calls urPlatformsGet, which then calls zeInit in the L0 adapter.

Reply:
I think it is OK for now to keep it here, but when we move away from PI (SYCL RT calls UR directly) it should be explicitly called from SYCL RT.